### ISSUE TITLE
Update Homebridge to 18.x from 16.x issue

### ISSUE THREAD

**Issue by timhubpie** [mwraith/homebridge-somfi-blinds#14]
Hi,

Worked perfect well for three years until I did a update to Homebridge 18.3 (including node upgrade to 20). 

‚This plugin threw an error from the characteristic 'Target Position': Unhandled error thrown inside write handler for characteristic: pigpio.gpioInitialise is not a function. See https://homebridge.io/w/JtMGR for more info.‘

Any idea?

Mike

**Comment by mwraith** [comment]
@timhubpie I am running the 1.8.4 of homebridge with same version of node and it's ok. You are still running homebridge as root user right?

Any other errors in the log with debug mode on?

**Comment by timhubpie** [comment]
MW, 
Many thanks for the quick reply. 
I mixed up the version, so running on Node.js Version v20.17.0 and 1.8.4 and somfy v0.4.1., debug mode on. 
It should be root, in terminal root@Raspberry:/var/lib/homebridge# 

After pushing in Home the 'LR Shutter Rear' down the following message in red applies: 'This plugin threw an error from the characteristic 'Target Position': Unhandled error thrown inside write handler for characteristic: pigpio.gpioInitialise is not a function. See https://homebridge.io/w/JtMGR for more info.'
Debug Log:
----
[9/20/2024, 2:19:36 PM] [LR Shutter Rear] Initializing Somfy Blinds accessory...
[9/20/2024, 2:19:36 PM] [LR Shutter Rear] Initializing child bridge 0E:C6:E6:A4:C6:09
[9/20/2024, 2:19:41 PM] [LR Shutter Rear] Launched child bridge with PID 21763
[9/20/2024, 2:19:43 PM] [LR Shutter Rear] Loaded homebridge-somfi-blinds v0.4.1 child bridge successfully
[9/20/2024, 2:19:43 PM] Loaded 11 cached accessories from cachedAccessories.0EEAA3FD8377.
[9/20/2024, 2:19:43 PM] Loaded 0 cached accessories from cachedAccessories.0EC6E6A4C609.
---
Child bridge is running, shows 'child_bridge.label_child_restart_plural', other plugins show 'Restart Child bridge. Maybe the multiple child bridges are the issue?

As I wrote no issues with 16.x.x (I forgot what the last version was) and the 18.7 note.

Tim
